tests/numpy/test_stack.py

```diff
@@ -91,6 +91,69 @@ def test_atleast_3d(nps_app_inst):
 
     try_multiple_nd(np.atleast_3d, nps.atleast_3d)
 
+
+def test_hstack(nps_app_inst):
+    import nums.numpy as nps
+    assert nps_app_inst is not None
+
+    a1 = nps.array((1, 2, 3))
+    a2 = np.array((1, 2, 3))
+    b = np.array((2, 3, 4))
+    assert np.allclose(nps.hstack((a1, b)).get(), np.hstack((a2, b)))
```

Comment:
Can we please test any auto-conversion functionality separately? Here, a nums array for `b` should be used instead of the numpy array.